### PR TITLE
Add languages to metadata and use for language integration in macOS

### PR DESCRIPTION
**Pull request by sdassow**
### Description:
Recent macOS versions offer support for selecting a language per application, and this PR is working towards integrating with that.

This is the first step to solve #5277 and this PR makes the language drop-down available, but the app itself does not seem to get it yet, so there seems to be more to it.

### Checklist:

- [X] Tests included.
- [X] Lint and formatter run with no errors.
- [X] Tests all pass.

### CODE REVIEW

**Review by andydotxyz — CHANGES_REQUESTED** [review]
Thanks so much for this nice new feature - but check out bundle.go changes before pushing!

File: cmd/fyne/internal/templates/bundled.go [hunk 1]
```diff
@@ -5,6 +5,11 @@ package templates
 
 import "fyne.io/fyne/v2"
 
+var resourceInfoPlistSwp = &fyne.StaticResource{
```
[the remark this answers is not in the document]
Comment (sdassow) [reply cmd/fyne/internal/templates/bundled.go:8]: Oops, I should indeed! Getting used my generators filtering temp files. Sorry, fixing...

[the remark this answers is not in the document]
Comment (sdassow) [reply cmd/fyne/internal/templates/bundled.go]: Better now, thanks!

**Review by andydotxyz — APPROVED** [review]
Sweet thanks

**Pull request merged**